packaging/prover-proxy/miden-prover-proxy.service

```diff
@@ -0,0 +1,13 @@
+[Unit]
+Description=Miden delegated prover proxy
+Wants=network-online.target
+
+[Install]
+WantedBy=multi-user.target
+
+[Service]
+Type=execExecStart=/usr/bin/miden-proving-service start-proxy --port 50051 63.176.73.108:50052 --status-port 8080
```

Comment:
This hard-codes IP address of the worker, right? How do we handle changes in the IP address or multiple deployments?

Reply:
For now I will have the VM instances update this with initialization scripts that modify the service file. But need to think of an appropriate solution for this in general.